communication/slack-config/channels.yaml

```diff
@@ -4,6 +4,8 @@
 channels:
   - name: airflow-operator
   - name: announcements
+  - name: antrea
```

Comment:
```suggestion
  - name: antrea-users
```

Reply:
Small nit, but the general convention we have when splitting up channels to users/dev like this is to add the `-users` suffix for the user channel. 

Also -- more a comment than a blocker, but you may want to consider a single channel if you don't expect a large number of people; just to more easily engage with a single group of folk.

Reply:
The yaml was about half and half with regards to the `-users` suffix so I wasn't sure which way to go.

I think it's fine on our side to start with a single channel, but in this case it seems that `antrea` (rather than `antrea-dev`) would be more appropriate?

Reply:
@mrbobbytables I amended my commit to have a single `antrea` channel

Reply:
yup 👍 will approve shortly.